cookbooks/aws-parallelcluster-common/resources/network_service/partial/_restart_network_alinux_centos_redhat.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+action :restart do
+  log "Restarting '#{network_service_name}' service, platform #{node['platform']} '#{node['platform_version']}'"
+
+  service network_service_name do
```

Comment:
**Nitpick.** Looks like `:restart` action is common for all the OSes.
The only difference is that, in `:reload`, ubuntu OSes apply network configuration after restarting.
This is not immediately clear from the code.
I would suggest to have a common partial with `:restart` action (and maybe the default behavior for `:reload`) and a partial for `:reload` specific for ubuntu OSes.

Reply:
Changed, good point, thanks!